### DISCUSSION TITLE
How many bounding boxes per class?

### DISCUSSION THREAD

**Discussion by arun-rangarajan** [ultralytics/yolov5#2024] [Q&A]
Thanks for your awesome project! We got Yolo v5 working for our use case.

We have very small objects (sometimes just 40x40 in a 1700x2200 img) and 20 different classes. Some classes have only a few hundred example bounding boxes and do not perform as well as others that have few thousand examples. What's the minimum number of bounding boxes needed per class?

**Comment by glenn-jocher** [comment]
There's no minimum number of instances.

For reference COCO has about 10k instances and 1k images per class, so if you're looking for comparable performance you should use a comparable quantity of data.
